lib/thrift/protocol/binary.ex

```diff
@@ -105,7 +106,7 @@ defmodule Thrift.Protocol.Binary do
   end
   def serialize(:application_exception, %TApplicationException{message: message, type: type}) do
     <<@string::size(8), 1::16-signed, byte_size(message)::size(32), message::binary,
-      @i32::size(8), 2::16-signed, type::32-signed, 0::size(8)>>
+      @i32::size(8), 2::16-signed, type::32-signed, @stop::8-signed>>
```

Comment:
It's always read and written as an `i8` (the same as a field type), but as it's always going to be 0 anyway, I like the idea of just writing `@stop` for the sake of brevity.